crates/ty_python_semantic/resources/mdtest/glossary.md

```diff
@@ -0,0 +1,84 @@
+# Glossary
```

Comment:
Documents like these tend to stagnate over time, so I'm also fine moving this content somewhere else and less central. But I think there would be room for more things that we could explain here. For example:
* Definition vs bindings vs declaration
* Place vs symbol vs qualified types vs types
* "Symbol" in semantic index building vs "symbol" in type inference